dev/centos-stream8/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+FROM docker.io/vespaengine/vespa-build-centos-stream8:latest
+
+ENV container docker
+
+# Fixes a problem with Rpmdb checksum cache conflicts: https://github.com/CentOS/sig-cloud-instance-images/issues/15
+RUN dnf -y install dnf-plugin-ovl
```

Comment:
Consider installing `dnf-plugin-ovl` earlier, i.e. in build image.

Reply:
Fixed.